scripts/email_customer_survey.py

```diff
@@ -0,0 +1,135 @@
+import sys
```

Comment:
No, can I remove it? It works but doesn't use your REST API module and is not of user for other reasons.